modules/using-oci-containers-for-model-storage.adoc

```diff
@@ -0,0 +1,209 @@
+:_module-type: PROCEDURE
+
+[id="using-oci-containers-for-model-storage_{context}"]
+= Using OCI containers for model storage
+
+[role='_abstract']
+
+As an alternative to storing a model in an S3 bucket or URI, you can upload models to OCI containers. Using OCI containers for model storage can help you:
+
+* Reduce start up times by avoiding downloading the same model multiple times.
+* Reduce disk space usage by reducing the number of models downloaded locally.
+* Improve model performance by allowing pre-fetching images.
+
+This guide shows how to manually deploy a MobileNet v2-7 model in an ONNX format, stored in an OCI image on an OpenVINO model server.
+
+ifndef::upstream[]
+[IMPORTANT]
+====
+ifdef::self-managed[]
+Using OCI containers for model storage is currently available in {productname-long} {vernum} as a Technology Preview feature.
+endif::[]
+ifdef::cloud-service[]
+Using OCI containers for model storage is currently available in {productname-long} as a Technology Preview feature.
+endif::[]
+Technology Preview features are not supported with Red{nbsp}Hat production service level agreements (SLAs) and might not be functionally complete.
```

Comment:
Suggestion: Replace Red {nbsp} Hat with the {org-name} attribute.

Reply:
```suggestion
Technology Preview features are not supported with {org-name} production service level agreements (SLAs) and might not be functionally complete.
```